_data/navigation.yml

```diff
@@ -123,7 +123,7 @@ for_developers:
         url: https://coverity.cern.ch/login/login.htm
         target: _blank
       - title: GitWeb
-        url: https://root.cern.ch/gitweb/?p=root.git;a=summary
```

Comment:
While you're at it: can you remove the gitweb entry altogether, please?

Reply:
done!